docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+---
+id: k8s_guide
+title: A Kubernetes tutorial
+---
+# Getting Started
+:::warning[DISCLAIMER]
+This tutorial contains some parts that are CERN-specific. <br/>
+The tools utilised to set up and to run the cluster are chosen accordingly. Feel free to use your own.
+:::
+
+In order to efficiently manage the cluster and implement continuous delivery, the following tools are required.
+
+## Kubernetes: `kubectl`
+[Reference Documentation](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/).
+## Helm Charts
+[Reference Documentation](https://helm.sh/docs/intro/install/#from-script).
+```bash
+curl -fsSL -o get_helm.sh https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3
+
+chmod 700 get_helm.sh
+
+./get_helm.sh
+```
+
+Also, add the `bitnami` repo: 
+```helm
+helm repo add bitnami https://charts.bitnami.com/bitnami
+```
+
+So that we can search the repo with `helm search repo bitnami`.
+
+## Continuous delivery: `flux`
+[Reference Documentation](https://fluxcd.io/flux/get-started/).
+### [Install the Flux CLI](https://fluxcd.io/flux/installation/#install-the-flux-cli)
+```sh
+curl -s https://fluxcd.io/install.sh | sudo bash
+git status
+git add .
+git commit -m "first commit"
+git push
+```
+### [Bootstrap Flux](https://fluxcd.io/flux/cmd/flux_bootstrap_gitlab/)
+Then setup the flux in the gitlab repo, by exporting the secret and running the command
+```sh
+flux bootstrap gitlab \
+  --owner=<group> \
+  --repository=<repository name> \
+  --branch=<main branch> \
+  --path=<path> \
+```
+## (optional) Monitoring: `k9s`
+Get the binary from [here](https://github.com/derailed/k9s/releases) looking for the proper dist; extract, and move to `/usr/local/bin/
+
+# Setting up the CERN infrastructure
+## Creating the cluster on Openstack
```

Comment:
A sentence about what each of these is used for would make it more applicable for non-cern people. (Eg: Cluster to deploy rucio, what the master node vs worker nodes do) so they know their facilities' equivalent

Reply:
Changed title, added a few lines explaining the purpose and usage of the master and worker nodes.